src/modding/ModManager.cs

```diff
@@ -419,6 +612,14 @@ private static bool IsAllowedModPath(string path)
         return true;
     }
 
+    private void UpdateTabTitles()
+    {
+        // The tab title has to be set here as they are normally set by node name in TabContainer
+        // Which means they won't be translated at all
```

Comment:
Is this a Godot engine bug? If the tab names with strings like "MOD_LOADER_TAB" won't get translated in the tab control, this is a Godot bug then and we should open an issue in our repo and link that here to keep track of when the engine issue is solved.

Reply:
I am not sure if it it's a bug or not, it could be intentional since node names should probably not get renamed by Godot at runtime but I am also not familiar enough with Godot to confidently say that.

Reply:
I can't really remember, but my point probably was that for display in the GUI Godot already translates a bunch of things automatically (for example setting the label text to a translation key shows the translated text instead of the literal data that is set in the text property). So it seems weird to me that Godot doesn't support translated text in tab names like it does on buttons, labels etc.

Reply:
It's definitely a bug since I just created a test project and the tabs are automatically translated when I was using gdscript in my minimal reproduction project. I try to submit the issue once I figure out how to reproduce the bug.

Reply:
Ok maybe I was an idiot or they fixed it in one of the version but translation works now... I guess I can get rid of it now